spec/locale.html

```diff
@@ -347,8 +347,9 @@ <h1>get Intl.Locale.prototype.region</h1>
         1. Assert: _locale_ can be matched by the <code>unicode_locale_id</code> Unicode locale nonterminal.
         1. Let _lang_ be the longest prefix of _locale_ matched by the <code>unicode_language_id</code> Unicode locale nonterminal.
         1. Let _subtags_ be a List of the subtags of _lang_.
-        1. If the length of _subtags_ is greater than 1 and the length of _subtags_[1] is 4, let _i_ be 2. Otherwise, let _i_ be 1.
-        1. If the length of _subtags_ is greater than _i_ and the length of _subtags_[_i_] is in the inclusive range from 2 to 3, return _subtags_[_i_].
+        1. NOTE: A <code>unicode_region_subtag</code> subtag is only valid immediately after an initial <code>unicode_language_subtag</code> subtag that itself might be followed by a <code>unicode_script_subtag</code> subtag, and <code>unicode_region_subtag</code> cannot be confused with <code>unicode_variant_subtag</code> because their productions are disjoint.
```

Comment:
This wording feels a little unclear to me. What about:

1. NOTE: A <code>unicode_region_subtag</code> subtag is only valid immediately after an initial <code>unicode_language_subtag</code>, or immediately after a <code>unicode_script_subtag</code> that is itself immediately after an initial <code>unicode_language_subtag</code>

Reply:
Acknowledged, and tweaked slightly.